src/ripple/protocol/TxFormats.h

```diff
@@ -57,6 +57,11 @@ enum TxType
     ttFEE               = 101,
 };
 
+inline bool isPseudoTx(TxType t)
+{
+    return t == ttAMENDMENT || t == ttFEE;
+}
```

Comment:
Is it possible that [with that change](https://github.com/ximinez/rippled/commit/06de64d68c40af5d114b3a82e16170f07f518830#diff-e29a42283d218be546efb2410c71ef51R170), the caller passes in an `STObject` that lacks a `sfTransactionType` field?  That was one reason I had gone the other way, making it explicit that a transaction type is required.